source/main.cpp

```diff
@@ -62,7 +59,7 @@ int main() {
                 delete globalCutscene;
                 globalCutscene = nullptr;
                 globalInGameMenu.show(false);
-                globalPlayer->setPlayerControl(true);
```

Comment:
why this change? I was about to merge this

Reply:
actually this wouldn't complie: _playerControl is private

Reply:
This was in the original code at the time